TGC/events/PORFlavor.txt

```diff
@@ -962,8 +962,8 @@ country_event = {
 
 	id =  49519
 	
-	title = "capital_change_kpb"
-	desc = "capital_change_kpb"
+	title = "EVTTITLE49519"
+	desc = "EVTDESC49519"
 	picture = "capital_change_kpb.dds"
 
 	trigger =  {  
```

Comment:
Suggest adding a fallback check that ensures KPB owns both Lisbon and Rio or has their capital in one of the two.

Like this:
```
tag = KPB
OR = {
  AND = {
    owns = (Lisbon's ID)
    owns = (Rio's ID)
  }
  AND = {
    capital = (Lisbon's ID)
    capital = (Rio's ID)
  }
}
```
In this order, to ensure optimization.